### ISSUE TITLE
Solenoid Valve not opening

### ISSUE THREAD

**Issue by adriandorrius** [Zer0-bit/gaggiuino#196]
Discord links not working so I'm opening issue here. 

everything was working and functional and even pulled a couple of shots. Did a little bit of wire management and then now the solenoid valve is not opening(ie. no water from group head). 

Behaviour: 

- Pump works when brew switch is flipped on 
- water comes out of steam wand when brew switch is flipped on 

Troubleshooting performed: 

- Cleaned solenoid valve and verified water is able to flow through the 2 very small holes 
- removed plunger inside valve and did a test run. water flows through all 3 outlets (grouphead, steam valve, drip tube)
- Used a multimeter to test the solenoid coil and got a non-zero, non-infinite resistance reading. 
- AC dimmer can regulate pump so it doesn't seem to be that. 
- Removed ac dimmer and rewired pump to original and still not working 

Basically down to the solenoid valve not getting the signal to open when brew switch is flipped. Not sure how to further troubleshoot.

**Comment by adriandorrius** [comment]
steam switches wired incorrectly. closing.
